### PR TITLE
david

### PR DESCRIPTION
**Pull request by Zoopers**
noooooo

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-www.jjsteyn.co.za
```
Comment (drubin) [inline CNAME:1]: You need this file. its the thing that tells the website its found on www.jjsteyn.co.za

File: Gemfile [hunk 2]
```diff
@@ -1,2 +0,0 @@
-source 'https://rubygems.org'
```
Comment (drubin) [inline Gemfile:1]: What happened to these 2 files?  

These are the 2 files that tell other people which gems you are using. So when I check out the code it automatically installs the required libs. :)

http://bundler.io/

**Comment by Zoopers** [comment]
Like I said, I'm completely lost. All the tutorials only tell you how to build a Jekyll site from scratch and Jekyll clears the github repository before it builds a new site, so it deleted those files.

**Comment by Zoopers** [comment]
There's a pull request that gives you the missing files, but I don't know if it works if I do that.

File: .ruby-version [hunk 3]
```diff
@@ -1 +0,0 @@
-2.1.3
```
Comment (drubin) [inline .ruby-version:1]: Also this one is fairly important (at least for me on my mac) it says which ruby version to use.

**Comment by drubin** [comment]
Other than the minor comments I added about files that got deleted this is super awesome!

**Comment by drubin** [comment]
Now you have a blog!

**Pull request merged**